images/Win11-DevBox/build.pkr.hcl

```diff
@@ -86,39 +86,32 @@ build {
   provisioner "powershell" {
     elevated_user     = build.User
     elevated_password = build.Password
-    inline = [
-      // "choco install postman --yes --no-progress",
-      "choco install googlechrome --yes --no-progress",
-      "choco install firefox --yes --no-progress"
+    scripts = [
+      "${path.root}/../../provisioning_scripts/Chocolatey-packages-install-upgrade.ps1"
     ]
   }
 
+  provisioner "windows-restart" {
+    restart_timeout = "30m"
+    pause_before    = "1m"
+  }
+    
   provisioner "powershell" {
     elevated_user     = build.User
     elevated_password = build.Password
-    scripts = [
-      "${path.root}/../../scripts/Install-Git.ps1",
-      "${path.root}/../../scripts/Install-GitHub-CLI.ps1",
-      "${path.root}/../../scripts/Install-DotNet.ps1",
-      "${path.root}/../../scripts/Install-Python.ps1",
-      "${path.root}/../../scripts/Install-GitHubDesktop.ps1",
+     scripts = [
+      "${path.root}/../../provisioning_scripts/Install-configure-ruby-firewall.ps1",
```

Comment:
I think this can be just: `Configure-ruby-firewall.ps1`

Reply:
done